src/components/common/ConfirmModal.tsx

```diff
@@ -57,6 +63,7 @@ const ConfirmModal = ({
   body: string | JSX.Element;
   actionLabel: string;
   open: boolean;
+  isDeleteConfirmation: boolean;
```

Comment:
can we add a default value to this? probably `false`

Reply:
should i make this prop optional too then?

Reply:
yep - most of the time if a prop has a default value then it should be optional